src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #060b35;
+}
+
+p {
+  margin: 0;
+}
+
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00ACDC;
+}
+
+.product {
+  margin: 50px;
```

Comment:
anyway the `product` is BEM-block, so it's not allowed to add margin property to it

So just remove `product__box` and apply all its styles to `product`
if you want to add some space around the card - add styles to the parent element